src/app/features/home/home.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, inject, OnInit } from '@angular/core';
+import { Job } from '../jobs/models/job';
+import { JobService } from '../jobs/service/job.service';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrl: './home.component.scss',
+})
+export class HomeComponent implements OnInit {
+  jobs$!: Observable<Job[]>;
```

Comment:
Let the "!" if jobs$ is initialized in the ngOnInit

Reply:
@PrettyJuice Indeed, we are then sure that it will be non-null/not undefined because it is initialized on the ngOnInit. I was just not a fan of that "bang" operator because it is a way to tell the compiler to chill where as I wanted really it to be strict by initializing it on the constructor.